src/components/layout/data/hooks.js

```diff
@@ -84,24 +83,50 @@ export const useStylesForCustomBrandColors = (enterpriseConfig) => {
       .btn-brand-outline-${colorName}:focus:before {
         border-color: ${brandColors[colorName].regular.hex()} !important;
       }
-
       .bg-brand-${colorName} {
         background-color: ${brandColors[colorName].regular.hex()} !important;
       }
-
       .border-brand-${colorName} {
         border-color: ${brandColors[colorName].regular.hex()} !important;
       }
-
       .color-brand-${colorName} {
         color: ${brandColors[colorName].regular.hex()} !important;
       }
-
       .text-brand-${colorName} {
         color: ${brandColors[colorName].textColor.hex()} !important;
       }
     `),
   }));
 
+  styles.push({
+    key: 'general',
+    styles: (`
+      .btn-primary {
```

Comment:
I think you're right that we'll also want to consider the `.btn-primary-outline` and `.btn-brand-outline` class names as well (not sure if we use brand outline, but there's a definitely a spot of two with primary outline).

Reply:
So because the designers want outline to default to the paragon styling, would it be best to just remove all instances of outline styling from the hook?